static/app/components/events/eventCause.tsx

```diff
@@ -9,23 +9,25 @@ import {CauseHeader, DataSection} from 'sentry/components/events/styles';
 import {Panel} from 'sentry/components/panels';
 import {IconAdd, IconSubtract} from 'sentry/icons';
 import {t} from 'sentry/locale';
+import {
+  withCommitters,
+  WithCommittersProps,
+} from 'sentry/stores/Commiters/CommittersContext';
```

Comment:
our folders and files usually start with a lowercase letter...should we update it here?

Reply:
Ah yes. Let me create a new PR on top that renames this so it doesnt all get messed up in git